src/web/src/components/Schedule.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     class="schedule"
-    :style="{ height: totalHeight + 'px' }"
+    :style="{ height: totalVHeight + 'vh', 'min-height': minHeight + 'px' }"
```

Comment:
smol nit, perhaps we can use the js template strings like so:
```diff
- :style="{ height: totalVHeight + 'vh', 'min-height': minHeight + 'px' }"
+ :style="{ height: `${totalVHeight}vh`, 'min-height': `${minHeight}px` }"
```
wyt? (also can be tweaked @ [lines 39-42](https://github.com/YACS-RCOS/yacs.n/pull/448/files#diff-3b36822eec87cadead13937bf56766759546ea4f5eaca69b8e3aa3caeae3552eR39-R42))